### PR TITLE
Update some tests that are currently skipped, so they can run

### PR DESCRIPTION
**Pull request by brybacki**
Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

**What this PR does / why we need it**:
Update the remaining SKIPPED tests to reflect the desired behavior 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by brybacki** [comment]
just to see what kind of flakiness is present here
/test pull-kvp-functional-test

**Comment by brybacki** [comment]
/test pull-kvp-functional-test
checking if the same tests fail

**Comment by brybacki** [comment]
/test pull-kvp-functional-test

looks good - check if it passes again

**Comment by brybacki** [comment]
About 11 tests uncommented and corrected, but a few still need some fixes.

File: tests/resource_filtering_test.go [hunk 1]
```diff
@@ -2409,7 +2412,7 @@ var _ = Describe("Resource excludes", func() {
 				By("Verifying included VM exists")
 				err = WaitForVirtualMachineStatus(*kvClient, namespace.Name, vmIncluded.Name, kvv1.VirtualMachineStatusStopped, kvv1.VirtualMachineStatusRunning)
 				Expect(err).ToNot(HaveOccurred())
-
+				time.Sleep(60 * time.Second)
```
Comment (awels) [inline tests/resource_filtering_test.go:2415]: We really want to wait 60 seconds?

**Review by awels — COMMENTED** [review]
I see many excludes, I am assuming this is expected?

**Comment by brybacki** [comment]
> I see many excludes, I am assuming this is expected?
About 6 tests still need improvements, I need to find an easy way to write something to disk and verify after backup

File: tests/resource_filtering_test.go (hunk 1 above)
Comment (brybacki) [reply tests/resource_filtering_test.go:2415]: no, that was a stop so I could do a troubleshooting

Comment (brybacki) [reply tests/resource_filtering_test.go]: removed

**Pull request merged**